app/auth/views/login_utils.py

```diff
@@ -14,7 +14,15 @@ def after_login(user, next_url):
     If user enables MFA: redirect user to MFA page
     Otherwise redirect to dashboard page if no next_url
     """
-    if user.enable_otp:
+    if user.fido_uuid is not None:
```

Comment:
(Optional) As `user.fido_uuid is not None` is frequently used, I wonder if it makes sense to a method like `User.fido_enabled` to wrap this logic. It seems quite useless for now but in the next step, we'll add something like `User.can_use_fido` as a way to beta-test this feature so the `fido_enabled()` can be written as:

```
def fido_enabled(self) -> bool:
     if not self.can_use_fido: return False
     if self.fido_uuid is not None: return True
     return False
```

Reply:
f7e33202421df53ca6f7ee09736c51153b3038a3